### DISCUSSION TITLE
is there an estimated release date for the current master branch?

### DISCUSSION THREAD

**Discussion by niXman** [jedisct1/libsodium#1150] [General]
Hi,

guys, is there an estimated release date for the current `master` branch?

best!

**Comment by jedisct1** [comment]
No 😀

**Comment by niXman** [comment]
I'm sad :(

**Comment by songgao-zoom** [comment]
@jedisct1 GitHub has a passive aggressive way to nudge you to change your mind :)

(Regardless, thanks for maintaining such a great and impactful project!)

https://user-images.githubusercontent.com/83841093/174352150-6f2c7938-de35-4dbd-ac25-85d8f0a98140.mov
